test/e2e/config/azure-dev.yaml

```diff
@@ -195,11 +195,6 @@ variables:
   CLUSTER_IDENTITY_NAME: "cluster-identity-sp"
   NODE_DRAIN_TIMEOUT: "60s"
   CI_VERSION: ""
-  # NOTE: INIT_WITH_BINARY and INIT_WITH_KUBERNETES_VERSION are only used by the clusterctl upgrade test to initialize
-  # the management cluster to be upgraded.
-  INIT_WITH_BINARY: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.5/clusterctl-{OS}-{ARCH}"
-  INIT_WITH_PROVIDERS_CONTRACT: "v1beta1"
-  INIT_WITH_KUBERNETES_VERSION: "v1.23.13"
```

Comment:
Where were these environment variables used? I don't see any references to them being removed.

Reply:
They were not used https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4433#discussion_r1445113631